sdk_container/src/third_party/coreos-overlay/app-emulation/hv-daemons/hv-daemons-9999.ebuild

```
@@ -1,32 +1,38 @@
-# Copyright 2044-2016 The Flatcar Maintainers
+# Copyright 2014-2025 The Flatcar Maintainers
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
-
-inherit coreos-kernel savedconfig systemd
-
-DESCRIPTION="HyperV guest support daemons."
+COREOS_SOURCE_REVISION=""
+inherit systemd
+
+DESCRIPTION="Hyper-V guest support daemons"
+HOMEPAGE="http://www.kernel.org"
+S="${WORKDIR}"
+LICENSE="GPL-2"
+SLOT="0"
 KEYWORDS="amd64 arm64"
 
+DEPEND="=sys-kernel/coreos-sources-${PV}${COREOS_SOURCE_REVISION:--r0}"
```

Could that maybe be:

```suggestion
DEPEND="~sys-kernel/coreos-sources-${PV}"
```

?